applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -34,11 +34,19 @@ config BT_AUDIO_PREF_SAMPLE_RATE_VALUE
 	default 0x08 if BT_AUDIO_PREF_SAMPLE_RATE_48KHZ
 
 choice BT_AUDIO_PREF_SAMPLE_RATE
-	prompt "Preferred audio sample rate"
+	prompt "Preferred BT audio sample rate"
```

Comment:
Nitpick: Maybe we can use LE audio instead of BT audio :P

Reply:
It's because the config is called BT_AUDIO_